app/views/admin/organisations/_people_order.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= render "govuk_publishing_components/components/summary_list", {
+  title:,
+  edit: {
+    href: "#",
+    link_text: sanitize("Reorder <span class=\"govuk-visually-hidden\">#{title}</span>"),
```

Comment:
i think

```
sanitize("Reorder tag.span(title, class"govuk-visually-hidden")),
```

is slighty cleaner. Wdyt?

Reply:
Yeah, for sure. We've made this change. ✅